deployment/terraform/storage.tf

```diff
@@ -1,29 +0,0 @@
-#
```

Comment:
I think that this is something we might want to keep. It'll setup a place for us to point the ALBs to to write their logs later.